filter/impl/access_log_filter.go

```diff
@@ -0,0 +1,211 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package impl
+
+import (
+	"os"
+	"reflect"
+	"strings"
+	"time"
+)
+
+import (
+	"github.com/apache/dubbo-go/common/constant"
+	"github.com/apache/dubbo-go/common/extension"
+	"github.com/apache/dubbo-go/common/logger"
+	"github.com/apache/dubbo-go/filter"
+	"github.com/apache/dubbo-go/protocol"
+)
+
+const (
+	//used in URL.
+	AccessLogKey      = "accesslog"
```

Comment:
in dubbo accessLogKey is dubbo.accesslog.
I recommand to change the key.

Reply:
It's the ACCESS_LOG_KEY, not the LOG_KEY. The difference is that the LOG_KEY is used to fetch the Logger. But the ACCESS_LOG_KEY is used to fetch the AccessLogFilter configuration from URL.
The code in Java is:
![image](https://user-images.githubusercontent.com/9923838/65419785-04192a80-de32-11e9-9323-4728a5107978.png)

![image](https://user-images.githubusercontent.com/9923838/65419838-21e68f80-de32-11e9-8c05-a26187d89d06.png)

we don't have the similar mechanism to fetch different logger. So I only need to define the ACCESS_LOG_KEY.

![image](https://user-images.githubusercontent.com/9923838/65419805-0da29280-de32-11e9-8bf6-a5b7e4108cdc.png)

![image](https://user-images.githubusercontent.com/9923838/65419820-198e5480-de32-11e9-97de-f438352342ee.png)